cmd/gossamer/genesis.go

```diff
@@ -77,6 +85,27 @@ func loadGenesis(ctx *cli.Context) error {
 	return t.Db().StoreGenesisData(gen)
 }
 
+// given raw genesis state data, return the initialized state trie and genesis block header.
+func initializeGenesisState(gen genesis.GenesisFields) (*trie.Trie, *types.Header, error) {
```

Comment:
the comment should start always with the method name for the godocs to work properly, eg: // initializeGenesisState

Reply:
unexported functions don't get included in the godoc, so I'm not sure if it makes a difference. I can change it anyways